changes/4617e683826ce812065e7ea82c3d1688.yaml

```diff
@@ -0,0 +1,8 @@
+---
+desc: Updated the ``Snap._applyNodeEdits()`` API to require a caller to provide the
+  ``user`` value in the ``meta`` argument. One remote API endpoint, ``ViewApi.saveNodeEdits()``,
+  previously allowed the ``meta`` argument to be provided without the ``user`` value.
+  This API now requires the ``user`` value is provided as well.
```

Comment:
Do you want to emphasize this as a breaking change?

Reply:
Noting that they're required now implies that its breaking? We can call it out stronger though if you think we should. I don't want this to be at the top of the changlog and making folks worried because it is largely an internal change which would affect someone who has written custom code to hand push edits directly on the View; which is not a regular usage pattern at all.